networkx/algorithms/broadcasting.py

```diff
@@ -0,0 +1,154 @@
+"""Routines to calculate the broadcast time of certain graphs.
+
+Broadcasting is an information dissemination problem in which a node in a graph,
+called the originator, must distribute a message to all other nodes by placing
+a series of calls along the edges of the graph. Once informed, other nodes aid
+the originator in distributing the message.
+
+The broadcasting must be completed as quickly as possible subject to the
+following constraints:
+- Each call requires one unit of time.
+- A node can only participate in one call per unit of time.
+- Each call only involves two adjacent nodes: a sender and a receiver.
+"""
```

Comment:
I also moved a big part of the docs for the ``tree_broadcast_center`` here since it seems like there was a requirement for docstrings for the module. I tried to imitate the example laid out in the boundary module of the algorithms. Please let me know if this looks ok on your end.

Reply:
I agree that we should add more examples. However the one you pointed to above seems a bit less formal to be included in the docs. I think the examples in the unit tests were quite good, maybe I can include some code in this part of the docs? Like the example below.
<img width="745" alt="image" src="https://github.com/networkx/networkx/assets/89562186/447a970a-1f3e-4b61-bf5d-2e1c51e8a8ed">